docs/sources/installation/_index.md

```diff
@@ -16,6 +16,9 @@ This section discusses the hardware and software requirements as well as the pro
 - [Install on macOS]({{< relref "mac" >}})
 - [Install on Windows]({{< relref "windows" >}})
 - [Run Docker image]({{< relref "docker" >}})
+- [Deploy on Kubernetes]({{< relref "kubernetes" >}})
 
 Refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}) topic. If you need to restart Grafana, refer to the [Restart Grafana]({{< relref "Restart-grafana.md" >}})
 topic for detailed information.
+
+If you want to install Grafana Enterprise, follow the instructions for installing Grafana on your particular operating system and then refer to [Activate an Enterprise license]({{< relref "activate-license.md" >}}). 
```

Comment:
The relref I'm using here is broken :/ 
I'm trying to figure out how to point to https://grafana.com/docs/grafana/latest/enterprise/activate-license/
which is under filepath docs/sources/enterprise/activate-license.md
but no luck. 

Can you help me understand what filepath our system is looking at when it does the build?

Reply:
Is this unrelated to the merge conflict?
`
Auto-merging docs/sources/installation/_index.md
--
14 | CONFLICT (content): Merge conflict in docs/sources/installation/_index.md`